exampleSite/content/about/content.md

```diff
@@ -4,12 +4,8 @@ fragment = "content-single"
 date = "2017-10-05"
 weight = 100
 #background = ""
-
-title = "About Syna"
-subtitle = ""
-
-
-content="""
+subtitle = "asdfasdfsaf"
```

Comment:
Not sure I agree with the choice for example data here ^^
Also title could be added or what was the reason to scrape it?

Reply:
LMFAO That was not supposed to be seen.
I'll readd title.